### ISSUE TITLE
An undefined type in a plugin type annotations does not provide a location

### ISSUE THREAD

**Issue by bartv** [inmanta/inmanta#1221]
A type used in an plugin annotation that does not exist gives an error without any location information:

```
Traceback (most recent call last):
  File "/home/bart/workspace/inmanta/src/inmanta/moduletool.py", line 592, in validate
    values = inmanta.compiler.do_compile()
  File "/home/bart/workspace/inmanta/src/inmanta/compiler/__init__.py", line 46, in do_compile
    success = sched.run(compiler, statements, blocks)
  File "/home/bart/workspace/inmanta/src/inmanta/execute/scheduler.py", line 206, in run
    self.define_types(compiler, statements, blocks)
  File "/home/bart/workspace/inmanta/src/inmanta/execute/scheduler.py", line 168, in define_types
    t.normalize()
  File "/home/bart/workspace/inmanta/src/inmanta/plugins.py", line 183, in normalize
    self.argtypes = [self.to_type(x[1], self.namespace) for x in self.arguments]
  File "/home/bart/workspace/inmanta/src/inmanta/plugins.py", line 183, in <listcomp>
    self.argtypes = [self.to_type(x[1], self.namespace) for x in self.arguments]
  File "/home/bart/workspace/inmanta/src/inmanta/plugins.py", line 273, in to_type
    return resolver.get_type(arg_type)
  File "/home/bart/workspace/inmanta/src/inmanta/ast/__init__.py", line 292, in get_type
    raise TypeNotFoundException(name, ns)
inmanta.ast.TypeNotFoundException: could not find type lcm::Service in namespace lcm
```

It should at least say that the error is caused while loading a plugin, preferably with the function and the location (source and line number)

**Comment by sanderr** [comment]
related to #1652

**Comment by sanderr** [comment]
@bartv I believe this might be fixed by #1763. I haven't worked with plugins before and I'm not sure what exactly you're referring to. Could you provide an example? If #1763 does fix it I'll add a test case to the PR.

**Comment by bartv** [comment]
A plugin that has the following definition:
```
def test(file: "std::WrongName") -> bool:
    return False
```
